### DISCUSSION TITLE
Inventory setups - Rune pouch (l) doesnt update runes in it

### DISCUSSION THREAD

**Discussion by alizan** [runelite/runelite#16161] [Q&A] [answered]
Hi,

I noticed Divine rune pouch (l) doesnt have a runes section when you add it in inventory setup, so you have to manually add the runes in additional filtered items.
Assume it is the same for the non-divine pouch as well if locked by trouver parchment.

**Comment by Nightfirecat** [comment, marked as answer]
Support for hub plugins should be directed to the author of the plugin.
You can find the support link by searching for the plugin in the Plugin Hub panel and clicking the `?` button on the plugin, or by right-clicking the plugin in the plugin panel and clicking the `Support` menu option.

For what it's worth, [I've already raised this as an issue on the plugin's issue tracker a few months ago](https://github.com/dillydill123/inventory-setups/issues/169).

**Reply by alizan** [reply]
Thanks for the info, looks like there were some other posts about this issue so hopefully the developer might upadte it
